seed/analysis_pipelines/better/buildingsync.py

```diff
@@ -79,10 +79,20 @@ def _build_better_input(analysis_property_view, meters):
     if property_state.property_type not in BETTER_TO_BSYNC_PROPERTY_TYPE:
         errors.append(
             f"BETTER analysis requires the property's type must be one of the following: {', '.join(BETTER_TO_BSYNC_PROPERTY_TYPE.keys())}")
+
+    valid_meters_and_readings = []
     for meter in meters:
-        for meter_reading in meter.meter_readings.all():
-            if meter_reading.reading is None:
-                errors.append(f'{meter}: MeterReading starting at {meter_reading.start_time} has no reading value.')
+        readings = meter.meter_readings.filter(reading__gte=1.0)
```

Comment:
make sure all readings we pass along are greater than or equal to 1 (BETTER requires this)

Reply:
Good look on that meter filter 🔥 .